packages/react-aria-components/src/Calendar.tsx

```diff
@@ -485,7 +490,11 @@ export {CalendarGridBodyForwardRef as CalendarGridBody};
 
 export interface CalendarCellProps extends RenderProps<CalendarCellRenderProps>, HoverEvents {
   /** The date to render in the cell. */
-  date: CalendarDate
+  date: CalendarDate,
+  /** The CSS [className](https://developer.mozilla.org/en-US/docs/Web/API/Element/className) for the element. A function may be provided to compute the class based on component state. */
+  cellClassName?: string | ((values: CalendarCellRenderProps & {defaultClassName: string | undefined}) => string),
```

Comment:
this has been asked for before, i forget where. right now there is no way to get styles or classnames on the td element of the table with all the days in it, so this adds that

Reply:
hmm this is weird. is there a way to avoid it?